images/capi/packer/config/ansible-args.json

```
@@ -1,3 +1,5 @@
+  "ansible_common_ssh_args": "-o IdentitiesOnly=yes -o HostKeyAlgorithms=+ssh-rsa -o PubkeyAcceptedAlgorithms=+ssh-rsa",
+  "ansible_scp_extra_args": "\"-O\""
```

Single quotes would also work, not sure which variant is actually more readable though.
```suggestion
  "ansible_scp_extra_args": "'-O'"
```

---

Changed